### ISSUE TITLE
Ancestor with mixed-type siblings produces wrong selector

### ISSUE THREAD

**Issue by psalaets** [tylerjpeterson/get-selector#1]
With this html

```html
<div>
  <h2>header</h2>
  <div>
    <span></span>
  </div>
</div>
```
the selector generated for the `span` is
`html > body > div:nth-child(1) > h2:nth-child(2) > span:nth-child(1)`
but should be
`html > body > div:nth-child(1) > div:nth-child(2) > span:nth-child(1)`

The top level `div` having two different element types is tripping it up.

**Comment by tylerjpeterson** [comment]
@psalaets - thanks for logging this. I believe 11a7078 should resolve (published on npm as `v1.0.1`).

**Comment by psalaets** [comment]
Cool, thanks for the quick turnaround
